includes/orders/functions/orders.php

```diff
@@ -1378,6 +1378,26 @@ function edd_get_net_order_statuses() {
 	return apply_filters( 'edd_net_order_statuses', $statuses );
 }
 
+/**
+ * Get the order status array keys which are considered recoverable.
+ *
+ * @since 3.0
+ * @return array An array of order status keys which are considered recoverable.
+ */
+function edd_recoverable_order_statuses() {
+	$statuses = array( 'pending', 'abandoned', 'failed' );
+	$statuses = apply_filters( 'edd_recoverable_payment_statuses', $statuses );
+
+	/**
+	 * Order statuses which are considered recoverable.
+	 *
+	 * @param $statuses {
+	 *        An array of order status array keys.
+	 * }
+	 */
+	return apply_filters( 'edd_recoverable_order_statuses', $statuses );
```

Comment:
I'm inclined to say we just keep the old `edd_recoverable_payment_statuses` filter for now and not introduce a new one. I know the naming isn't perfect with our new terminology, but in the event that all the parameters are still the same, I think keeping the old ones is simpler.